python/lib/cloudutils/serviceConfigServer.py

```diff
@@ -107,7 +107,8 @@ def checkHostName():
         bash("chown cloud.cloud /var/run/cloudstack-management.pid")
         #distro like sl 6.1 needs this folder, or tomcat6 failed to start
         checkHostName()
-        bash("mkdir /var/log/cloudstack-management/")
+        bash("mkdir /var/log/cloudstack/")
+        bash("mkdir /var/log/cloudstack/management/")
```

Comment:
I agree with @milamberspace .
btw, what about the /var/log/cloudstack/usage for usage server?